CHANGELOG.md

```diff
@@ -28,6 +28,95 @@ The changelog is generated during the release process using [ChangelogLinker](ht
 
 <!-- changelog-linker -->
 
+<!-- dumped content start -->
+
+## Unreleased
+
+### [shopsys/coding-standards]
```

Comment:
it would be nice if it was possible to specify the order of packages in the configuration, it is important to us to have `shopsys/framework` package on the first place, `shopsys/project-base` on the second place etc.

Reply:
Do you have config proposal + solution in code for this feature?

Reply:
hi, we discussed it and we can live with the default alphabetical sorting :slightly_smiling_face: